src/pages/resume.js

```diff
@@ -45,270 +48,145 @@ const Resume = () => (
       <section className="entry-list__item row">
         <div className="column">
           <span className="entry-list__meta">
-            <time dateTime="2015-10-16">Jan '16</time> &ndash;
+            <time dateTime="2015-10-16">April '18</time> &ndash;
             <time dateTime="2015-12-18">Present (Full-time employment)</time>
           </span>
         </div>
         <div className="column column-75">
           <h3 className="entry-list__heading">
-            <img src={canonicalLogo} alt="Canonical logo" />
+            <img src={citiLogo} alt="Citi logo" />
           </h3>
           <p>
-            <em>Canonical</em> is best known as the company that delivers
-            Ubuntu, an open source software platform that runs everywhere from
-            the smartphone to the cloud.
+            <em>Citi</em> is an American multinational investment bank and financial services corporation,
+            who provide consumers, corporations, governments and institutions 
+            with a broad range of financial services and products.
           </p>
           <p>
-            I'm currently working with Canonical to support their web offering
-            in the front end space across their multiple product streams.
-          </p>
-          <p>
-            Canonical&#039;s core mission to make open source software available
-            to people everywhere and one of the many perks of working with them
-            is the ability to contribute back to the open source community. One
-            highlight I'm currently working on is Vanilla Framework, a SASS
-            based framework of patterns to support all of Canonical&#039;s
-            upcoming web releases.
+          In my role as an Application Developer with Citi, 
+          I work with a development team to deliver continuous stability and enhancement to the e – Trading platform. 
```

Comment:
```suggestion
          I work with a development team to deliver continuous stability and enhancement to the e-Trading platform. 
```

I've removed the spaces around the hyphen here, does the official name internally definitely have a hyphen?

Reply:
I left the .svg in there as i thought you were going to do some magical work with them to resolve the image issue i was receiving